packages/thrift-client/src/main/connections/request.ts

```diff
@@ -32,11 +34,13 @@ export class RequestConnection<TClient> extends HttpConnection<TClient> {
         .post({
           body: dataToWrite,
           headers: {
-            'Content-length': dataToWrite.length,
+            'content-length': dataToWrite.length,
```

Comment:
I did not check the Axios connection but this may be set automatically on the Request connection. It is at least set on the request in the receiving server.

I receive connection and content-length.

Reply:
I see Content-length was set in the old version of the code so it may not have been Request.

Reply:
Yeah, case insensitive, but when using many node methods node normalizes to lower case, so I just do this for consistency.